LvivCompany.Bookstore/LvivCompany.Bookstore.Web/Startup.cs

```diff
@@ -30,7 +30,7 @@ public Startup(IHostingEnvironment env)
             Configuration = builder.Build();
         }
 
-        public IConfiguration Configuration { get; }
+        //public IConfiguration Configuration { get; }
```

Comment:
Keep your code clean. Remove code if it's not required, don't comment it. Use comments only for documentation purposes.